requirements.txt

```diff
@@ -16,8 +16,7 @@ rosdistro==0.6.2
 rosinstall-generator==0.1.13
 rospkg==1.1.1
 six==1.10.0
-trollius==2.1
+trollius==2.1.post2
```

Comment:
Pip for python3 could only find `2.1.post2`. This module is not technically supported for python 3, so frankly I'd like to remove it, but I left it in in the interests of minimizing changes.